.github/workflows/acceptance-tests.yml

```diff
@@ -54,17 +65,18 @@ jobs:
         RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_DEV_KUBECONFIG }}
         RENKUBOT_RANCHER_BEARER_TOKEN: ${{ secrets.RENKUBOT_RANCHER_BEARER_TOKEN }}
+        RANCHER_DEV_API_ENDPOINT: ${{ secrets.RANCHER_DEV_API_ENDPOINT }}
         RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
         RENKU_ANONYMOUS_SESSIONS: true
         RENKU_TESTS_ENABLED: true
         TEST_ARTIFACTS_PATH: "tests-artifacts-${{ github.sha }}"
-        renku_core: "@${{ github.head_ref }}"
+        renku_core: "@develop"
```

Comment:
Is this something temporary to test the CI deployment?

Reply:
no this is on purpose - it deploys the current `develop` branch and plugs this branch in as a custom image (see a few lines below)

Reply:
this branch cannot be deployed alone with the current deployment config and acceptance tests